src/shared-components/SidebarPluginSection/SidebarPluginSection.tsx

```
@@ -0,0 +1,139 @@
+        <h5
+          className={`${styles.titleHeader} text-secondary`}
+          style={{
+            fontSize: '1.1rem',
+            marginTop: '1.5rem',
+            marginBottom: '0.75rem',
+          }}
+          data-testid="pluginSettingsHeader"
+          aria-hidden={hideDrawer ? 'true' : undefined}
+        >
+          {!hideDrawer && tCommon('plugins')}
+        </h5>
```

Biome (2.1.2)

[error] Provide screen reader accessible content when using heading  elements.

All headings on a page should have content that is accessible to screen readers.

(lint/a11y/useHeadingContent)